src/get-started/java-binding.md

```diff
@@ -52,6 +51,9 @@ Follow these steps:
   ```
 
   <!-- cspell:enable -->
+  This step is not necessary during installation, but is required to configure
```

Comment:
Which this? The following or the last? If the last, then shan't we say that before?

Reply:
This is the last step. I can move to the beginning, I supposed you meant this in the previous comment.